param.h

```diff
@@ -3517,13 +3517,13 @@ Until then, just keep it different than DGEMM_DEFAULT_UNROLL_N to keep copy rout
 #define ZGEMM_DEFAULT_UNROLL_N  4
 #define ZGEMM_DEFAULT_UNROLL_MN  16
 
-#define SGEMM_DEFAULT_P 128
-#define DGEMM_DEFAULT_P 160
+#define SGEMM_DEFAULT_P 30
```

Comment:
How were the default P and Q chosen for 128KB cache?

Reply:
https://github.com/OpenMathLib/OpenBLAS/pull/4381 demonstrated values that worked well for a 1MB L2 cache, so I divided that by 8.

If you have a more scientific approach, I'd be happy to hear it 😸